pep/app/styles/_variables.scss

```diff
@@ -15,6 +15,7 @@ $progress-spinner-loading-size: 15px;
 $progress-spinner-loading-border-width-lg: 4px;
 $progress-spinner-loading-size-lg: 50px;
 $drag-bar-thickness: 5px;
+$information-bar-zindex: 99999999;
```

Comment:
can you move this to the `//Z-INDICES` group below, and rename it to follow the `$z-*` convention